src/apps/bridge/aanderaaController.cpp

```diff
@@ -46,6 +47,7 @@ typedef struct aanderaaControllerCtx {
   TimerHandle_t _sample_timer;
   BridgePowerController *_bridge_power_controller;
   cfg::Configuration *_usr_cfg;
+  uint32_t _transmit_aggregations;
```

Comment:
Why uint32 instead of bool? Are there more possible values? Does the config system not have a notion of bool?

Reply:
Let me check, I just based this on the bridge_power_controller's `bridgePowerControllerEnabled` which uses a uint32.

Reply:
No bool in the config data types
```
typedef enum ConfigDataTypes{
    UINT32,
    INT32,
    FLOAT,
    STR,
    BYTES,
} ConfigDataTypes_e;
```